bevy_asset_loader/examples/progress_tracking.rs

```diff
@@ -95,7 +102,10 @@ fn expect(
 
 fn print_progress(progress: Option<Res<ProgressCounter>>) {
     if let Some(progress) = progress {
-        info!("Current progress: {:?}", progress.progress());
+        // The condition to remove hell in the console
+        if progress.is_changed() {
+            info!("Current progress: {:?}", progress.progress());
```

Comment:
Now that we don't log every frame, could we add the current frame here? E.g. something along the lines of "[Frame x] Updated Progress: xyz".